src/dotnet/commands/dotnet-compile-native/ArgumentsParser.cs

```diff
@@ -1,154 +1,124 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.CommandLine;
 using System.Linq;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
 
```

Comment:
@MichalStrehovsky pointed out this may make it more difficult to move native to corert repo.

@piotrpMSFT @eerhardt, are you planning to make a `nupkg` similar to the `Utils` package, so when we move `dotnet-compile-native` to corert, we can consume this there?